### DISCUSSION TITLE
Fixing the range of the legend in plotEmbedding()

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#809] [General]
Hi,

 Is there some way to fix the legend scale when comparing different markers using plotEmbedding() across the same matrix type be it gene score or gene expression or Motif deviation?

I want the legend to show the exact same scale.

Thanks!

**Comment by rcorces** [comment]
The `...` allows pass through arguments to `ggPoint()`. So you can set the `colorLimits`.
https://www.archrproject.com/reference/plotEmbedding.html
This concept also applies to many other functions in ArchR so pay close attention to the `...` additional arguments elements.

**Comment by sylestiel** [comment]
That is definitely good to know.

Thanks!
